tests/test_constraints.py

```diff
@@ -178,7 +178,12 @@ def test_batched_constraints(backend):
     suggested_pars = [1.0] * 3 + [0.0] * 5  # 2 pois 5 norm
```

Comment:
Perhaps instead of wrapping `suggested_pars` multiple times in the tests

```
pyhf.tensorlib.astensor(suggested_pars)
```

instead could just do it once here?

```suggestion
    suggested_pars = pyhf.tensorlib.astensor([1.0] * 3 + [0.0] * 5)  # 2 pois 5 norm
```

Reply:
I'd prefer to keep it explicit